logs_analyzer/app/scala/src/test/scala/com/databricks/apps/logs/ApacheAccessLogTest.scala

```diff
@@ -0,0 +1,50 @@
+package com.databricks.apps.logs
+
+import org.scalatest.FlatSpec
+
+class ApacheAccessLogTest extends FlatSpec {
+
+  "parseLogLine" should "return empty value for null argument" in {
```

Comment:
Instead of writing assert everywhere, use triple equals (===) or the [matcher DSL](http://www.scalatest.org/user_guide/using_matchers)